docs/dev-guide/adding-grid-support/adding-grid-support-step-by-step-guide/add-grid-config.md

```diff
@@ -1,3 +1,175 @@
+
+## Grid Naming Conventions
+
+The atmosphere grid name should always indicate the "ne" value and add "pg2" to indicate that the physgrid is being used. For a regionally refined mesh (RRM) the grid name should always start with `ne0` followed a descriptive string that includes the region being refined and the degree of refinement.
+
+**Example**: `ne0np4_northamericax4v1`
+
+Note that the example indicates a `4x` refinement, but does indicate the base resolution, which is useful to know. A better grid name would be `ne0np4_northamerica30x4v1`, because this tells us that the grid is consistent with `ne30` in the unrefined regions.
+
+For a rectilinear lat-lon grid used by the land and/or river models the grid name should start with "r" and typically use spacing less than one degree, so they indicate the nominal grid spacing, starting with "0" and omitting the decimal.
+
+**Examples**: `r05` is 0.5 degree spacing and `r0125` is 1/8 or 0.125 degree spacing.
+
+For a mono-grid, which can only be used for idealized simulations such as aqua planet and RCE, the grid is written twice to indicate that both atmosphere/land and ocean/sea-ice models are on the same grid.
+
+**Example**: `ne30pg2_ne30pg2`
+
+Bi-grid options should indicate two different grids used for atmosphere/land and ocean/sea-ice models.
+
+**Example**: `ne30pg2_IcoswISC30E3r5`
+
+Tri-grid options should indicate three different grids used for atmosphere, land, and ocean/sea-ice models, with the land grid appearing in the middle.
+
+**Example**: `ne30pg2_r05_IcoswISC30E3r5`
+
+## Grid Definition
+
+### Adding a New Grid Alias
+
+Grid aliases are defined in specified in `cime_config/config_grids.xml` and are used to specify the grid for a case when calling `create_newcase` via the `--res` argument. Below is an example grid alias for the `ne30pg2_r05_IcoswISC30E3r5` grid used in E3SMv3 production simulations.
+
+```xml
+    <model_grid alias="ne30pg2_r05_IcoswISC30E3r5">
+      <grid name="atm">ne30np4.pg2</grid>
+      <grid name="lnd">r05</grid>
+      <grid name="ocnice">IcoswISC30E3r5</grid>
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>IcoswISC30E3r5</mask>
+    </model_grid>
+```
+
+### Domain Files
+
+Domain files are needed for each grid and are specified in the `<domains>` section of `cime_config/config_grids.xml`. The default domain files are grouped by the atmosphere grid. The section for the typical `ne30pg2` grid looks as follows:
```

Comment:
It would be good to have a sentence explaining why we need domain files in the first place.

Reply:
I wasn't confident in my understand of this to explain it adequately.

Reply:
Who does?!